src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -91,21 +91,21 @@ Before starting our [automated installer](https://one.newrelic.com/launcher/nr1-
   >
     To deploy the Kubernetes integration with [OpenShift](https://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG):
 
-    1. Add the `<release_name>-newrelic-infrastructure` service account to your privileged [Security Context Constraints](https://docs.openshift.com/enterprise/3.0/admin_guide/manage_scc.html):
+    1. Add the `<>{'<release_name>'}</>-newrelic-infrastructure` service account to your privileged [Security Context Constraints](https://docs.openshift.com/enterprise/3.0/admin_guide/manage_scc.html):
 
        ```
        oc adm policy add-scc-to-user privileged \
-       system:serviceaccount:<mark><namespace></mark>:<release_name>-newrelic-infrastructure
+       system:serviceaccount:<mark>&lt;namespace&gt;</mark>:<>{'<release_name>'}</>-newrelic-infrastructure
```

Comment:
I am not entirely surre about this one. Is there any way to check if this is rendering correctly?

Reply:
Hi @roobre There is a way to check this. That's to build the project locally. I'll do that, and share a screenshot with you, but for future reference, this section of the contributor guide describes how to view your changes locally:
https://github.com/newrelic/docs-website/blob/develop/CONTRIBUTING.md#local-development